contracts/src/v0.8/ccip/package.json

```diff
@@ -0,0 +1,92 @@
+{
+  "name": "@chainlink/contracts-ccip",
+  "version": "1.5.0",
+  "description": "Chainlink smart contracts for CCIP",
+  "author": "Chainlink devs",
+  "license": "BUSL-1.1",
+  "private": false,
+  "scripts": {
+    "test": "hardhat test --parallel",
+    "size": "hardhat size-contracts",
+    "clean": "hardhat clean",
+    "compile:native": "./../../../scripts/native_solc_compile_all",
+    "compile": "hardhat compile",
+    "coverage": "hardhat coverage",
+    "prepublishOnly": "pnpm compile && ./../../../scripts/prepublish_generate_abi_folder",
+    "publish-beta": "pnpm publish --tag beta",
+    "publish-prod": "pnpm publish --tag latest",
+    "solhint": "solhint --config ./../../../.solhint.json --max-warnings 0 \"./**/*.sol\"",
+    "solhint-test": "solhint --config \"./../../../.solhint-test.json\"  --ignore-path \"./../../../.solhintignore-test\" --max-warnings 0 \"./**/*.sol\""
+  },
+  "files": [
+    "LICENSE.md",
+    "*.sol",
+    "**/*.sol",
+    "abi/v0.8"
```

Comment:
@RensR would we need a path to ABI here? Which path should it be, the parent API path or just a specific dir from there?